docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -0,0 +1,437 @@
+---
+aliases:
+  - /docs/grafana/latest/dashboards/dashboard-manage/
+  - /docs/grafana/latest/features/dashboard/dashboards/
+  - /docs/grafana/latest/dashboards/dashboard-folders/
+  - /docs/grafana/latest/reference/dashboard_folders/
+  - /docs/grafana/latest/dashboards/export-import/
+  - /docs/grafana/latest/reference/export_import/
+  - /docs/grafana/latest/troubleshooting/troubleshoot-dashboards/
+  - /docs/grafana/latest/dashboards/time-range-controls/
+  - /docs/grafana/latest/reference/timerange/
+  - /docs/grafana/latest/dashboards/scripted-dashboards/
+  - /docs/grafana/latest/reference/scripting/
+  - /docs/grafana/latest/panels/working-with-panels/organize-dashboard/
+title: Manage dashboards
+menuTitle: Manage dashboards
+weight: 8
+eywords:
+  - grafana
+  - dashboard
+  - dashboard folders
+  - folder
+  - folders
+  - import
+  - export
+  - troubleshoot
+  - time range
+  - scripting
+---
+
+# Manage dashboards
+
+A dashboard is a set of one or more [panels]({{< relref "../../panels/" >}}) that visually presents your data in one or more rows.
+
+For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
+
+This topic includes techniques you can use to manage your Grafana dashboards, including:
+
+- [Creating and managing dashboard folders](#create-and-manage-dashboard-folders)
+- [Exporting and importing dashboards](#export-and-import-dashboards)
+- [Configuring dashboard time range controls](#configure-dashboard-time-range-controls)
+- [Organizing dashboards](#organize-a-dashboard)
+- [Troubleshooting dashboards](#troubleshoot-dashboards)
+
+When managing dashboads, you can use:
```

Comment:
@torkelo , this is legacy content that for me doesn't really belong, but wanted your opinion on if we should remove it.

Reply:
removed.